roles/kubernetes/control-plane/vars/main.yaml

```diff
@@ -6,3 +6,4 @@ kube_apiserver_admission_plugins_needs_configuration:
 - ImagePolicyWebhook
 - PodSecurity
 - PodNodeSelector
+- ResourceQuota
```

Comment:
@VannTen Would you prefer the following to replace this line?  This is what I had before.
```
- "{{ (kube_resource_quota_limited_resources is defined and (kube_resource_quota_limited_resources | length > 0) ) | ternary('ResourceQuota','') }}"
```
 It's far uglier, but prevents the file from being templated.

Reply:
I don't mind. The plugins needs to be in the `_enabled`  currently anyway, so let's keep this. If we refactor this in the future we'll revisit